service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/controller/helpers/CreateResponseEntityCreator.java

```diff
@@ -36,7 +36,7 @@ public ResponseEntity<O> create(O serviceResponse) {
         String responseIdentifier = "0";
 
         if (serviceResponse != null) {
-            String identifier = serviceResponse.getIdentifier();
+            String identifier = serviceResponse.getId();
```

Comment:
I think this should be changed to id, also to reflect getId

Reply:
Done